client/packages/invoices/src/Returns/modals/OutboundReturn/ReturnSteps.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  useTranslation,
+  WizardStepper,
+  TabPanel,
+  TabContext,
+  OutboundReturnLineNode,
+  RecordPatch,
+} from '@openmsupply-client/common';
+import { QuantityToReturnTable } from './ReturnQuantitiesTable';
+import { ReturnReasonsTable } from '../ReturnReasonsTable';
+
+export enum Tabs {
+  Quantity = 'Quantity',
+  Reason = 'Reason',
+}
+
+interface ReturnStepsProps {
+  currentTab: string;
+  lines: OutboundReturnLineNode[];
+  update: (patch: RecordPatch<OutboundReturnLineNode>) => void;
+}
+
+export const ReturnSteps = ({
+  currentTab,
+  lines,
+  update,
+}: ReturnStepsProps) => {
+  const t = useTranslation('replenishment');
+
+  const returnsSteps = [
+    { tab: Tabs.Quantity, label: t('label.select-quantity'), description: '' },
+    { tab: Tabs.Reason, label: t('label.reason'), description: '' },
+  ];
+
+  const getActiveStep = () => {
+    const step = returnsSteps.findIndex(step => step.tab === currentTab);
+    return step === -1 ? 0 : step;
+  };
+
+  return (
+    <TabContext value={currentTab}>
+      <WizardStepper activeStep={getActiveStep()} steps={returnsSteps} />
```

Comment:
Shall I make a new issue to add some kind of linking feature to this stepper? So you could click on previous steps to go back to them?

Reply:
Yes, definitely, I naturally tried to go back and realised I couldn't! 

I think a "Back" button is probably enough, but we should replicate the UX of other steppers.

Reply:
Yeah cool - I'll make a note to do this separately too... we'd need to update our common `Modal` component to support another type of button